rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -479,10 +583,10 @@ private Stream<Arguments> expectedSizeArgs() {
         @Tag("getActiveFederationThreshold")
         @MethodSource("expectedThresholdArgs")
         void getActiveFederationThreshold_returnsExpectedThresholdAccordingToActivationAgeAndActivations(
-            long currentBlock, 
-            ActivationConfig.ForBlock activations, 
+            long currentBlock,
+            ActivationConfig.ForBlock activations,
             int expectedThreshold) {
-            
+
```

Comment:
i dont know how te get rid of this indentation changes. I tried but I couldn't

Reply:
I think this looks good